client.go

```diff
@@ -98,6 +98,10 @@ func runCommandInContainer(img string, cmd []string) bool {
 		if res.Error != nil {
 			log.Println(res.Error)
 		} else {
+			if err != nil {
```

Comment:
Why ? The proper error is res.Error, which has already been handled. As far as I can tell, this `err` is the one from the call of `startContainer` which has already been handled no ?